graylog2-server/src/main/java/org/graylog/events/notifications/EventNotificationExecutionJob.java

```diff
@@ -130,12 +132,14 @@ public JobTriggerUpdate execute(JobExecutionContext ctx) throws JobExecutionExce
         } catch (TemporaryEventNotificationException e) {
             throw new JobExecutionException(
                     String.format(Locale.ROOT, "Failed to execute notification, retrying in %d minutes - <%s/%s/%s>",
-                            retryTimeInMinutes,
+                            eventsConfiguration.eventNotificationsRetry().toStandardDuration().getStandardMinutes(),
                             notification.id(),
                             notification.title(),
                             notification.config().type()),
                     trigger,
-                    ctx.jobTriggerUpdates().retryIn(retryTimeInMinutes, TimeUnit.MINUTES),
+                    ctx.jobTriggerUpdates().retryIn(
+                        eventsConfiguration.eventNotificationsRetry().toStandardDuration().getStandardMinutes(),
+                        TimeUnit.MINUTES),
```

Comment:
Using MINUTES here doesn't allow users to use a retry under 1 minute. I think we should use MILLISECONDS here to make it more flexible.